numpy/core/fromnumeric.py

```diff
@@ -797,9 +797,10 @@ def sort(a, axis=-1, kind='quicksort', order=None):
     =========== ======= ============= ============ =======
        kind      speed   worst case    work space  stable
     =========== ======= ============= ============ =======
-    'quicksort'    1     O(n^2)            0          no
-    'mergesort'    2     O(n*log(n))      ~n/2        yes
-    'heapsort'     3     O(n*log(n))       0          no
+    'quicksort'    1     O(n^2)            0         no
```

Comment:
hm unrelated, but this worstcase is wrong as we are using an introsort with nlogn worstcase